fastlane/Fastfile

```diff
@@ -18,24 +18,27 @@ ENV_FILE_NAME = '.wpios-env.default'
 USER_ENV_FILE_PATH = File.join(Dir.home, ENV_FILE_NAME)
 SECRETS_DIR = File.join(Dir.home, '.configure', 'wordpress-ios', 'secrets')
 PROJECT_ENV_FILE_PATH = File.join(SECRETS_DIR, 'project.env')
+APP_STORE_CONNECT_KEY_PATH = File.join(SECRETS_DIR, 'app_store_connect_fastlane_api_key.json')
 
 # Other defines used across multiple lanes
 REPOSITORY_NAME = 'WordPress-iOS'
 
 import 'Jetpack-Fastfile'
 
-ALL_BUNDLE_IDENTIFIERS = [
-  'org.wordpress',
-  'org.wordpress.WordPressShare',
-  'org.wordpress.WordPressDraftAction',
-  'org.wordpress.WordPressTodayWidget',
-  'org.wordpress.WordPressStatsWidgets',
-  'org.wordpress.WordPressNotificationServiceExtension',
-  'org.wordpress.WordPressNotificationContentExtension',
-  'org.wordpress.WordPressAllTimeWidget',
-  'org.wordpress.WordPressThisWeekWidget',
-  'org.wordpress.WordPressIntents'
-]
+APP_STORE_VERSION_BUNDLE_IDENTIFIER = 'org.wordpress'
+EXTENSIONS_BUNDLE_IDENTIFIERS = %w[
+  WordPressShare
+  WordPressDraftAction
+  WordPressTodayWidget
+  WordPressStatsWidgets
+  WordPressNotificationServiceExtension
+  WordPressNotificationContentExtension
+  WordPressAllTimeWidget
+  WordPressThisWeekWidget
+  WordPressIntents
+].map { |suffix| "#{APP_STORE_VERSION_BUNDLE_IDENTIFIER}.#{suffix}" }
+
+ALL_BUNDLE_IDENTIFIERS = [APP_STORE_VERSION_BUNDLE_IDENTIFIER] + EXTENSIONS_BUNDLE_IDENTIFIERS
```

Comment:
btw, just FYI and to improve your ruby knowledge about the splat operator that I talked about in the other comment, know that you could have used the splat operator in a similar way here to transform the `EXTENSIONS_BUNDLE_IDENTIFIERS` array into a list of values… and put that at the end of the array like if you inserted that list literally. i.e.:

`ALL_BUNDLE_IDENTIFIERS = [APP_STORE_VERSION_BUNDLE_IDENTIFIER, *EXTENSIONS_BUNDLE_IDENTIFIERS]`

Not saying that you should do that – I think both ways of assembling the final array are totally fine 👍  – but just thought you'd be interested and curious about this to improve your ruby knowledge 🙂

Reply:
Thank you for that. I wasn't happy with the look of it or with the unnecessary creation of an Array just for `APP_STORE_VERSION_BUNDLE_IDENTIFIER`.

I'm disappointed it didn't occur to me to try and use the splat which you already mentioned 🙃

I'll definitely adopt this as I keep working in this area